kubeflow/common/kubeflow-namespace/Kptfile

```diff
@@ -0,0 +1,13 @@
+apiVersion: kpt.dev/v1alpha1
+kind: Kptfile
+metadata:
+  name: profiles
+packageMetadata:
+  shortDescription: profile controller config.
```

Comment:
Nit: update package name and description

Reply:
Thank you! Applied.